HW8(MVC)/scripts/Model.js

```diff
@@ -0,0 +1,37 @@
+// NOT INTERACTING WITH A CONTROLLER. SEEMS LIKE A MISTAKE.
+class Model {
+  constructor() {}
+
+  addTask(title) {
+    const currentTasks = this.getTasksList();
+
+    currentTasks.push({
+      'title': title,
+      'isCompleted': false
+    });
+
+    this.setTasksList(currentTasks);
+  }
+
+  getTasksList() {
+    const tasks = JSON.parse(localStorage.getItem('savedTasks')) ?? { list: [] };
```

Comment:
'savedTasks' - better save in private property

Reply:
Same as above. How exactly should I implement "private"?